index.js

```diff
@@ -1,32 +1,49 @@
 var express = require('express');
 var bodyParser = require('body-parser');
+var dataStore = require('nedb');
 
 var port = 3000;
 var BASE_API_PATH = "/api/v1";
-var contacts = [
-{"name" : "peter", "phone" : "12456"},
-{"name" : "john", "phone" : "23213"}    
-];
+var DB_FILE_NAME = __dirname + "/contacts.json";
 
 console.log("Starting API server...")
 
 var app = express();
 app.use(bodyParser.json());
 
+var db = new dataStore({
+    filename: DB_FILE_NAME,
+    autoload: true
+})
+
 app.get("/", (req, res) => {
     res.send ("<html><body><h1>My server</h1></body></html>");
 });
 
 app.get(BASE_API_PATH + "/contacts", (req, res) => {
     console.log(Date() + "- GET /contacts");
-    res.send(contacts);
+    db.find({}, (err,contacts) => {
+        if (err) {
+            console.log(Date() + " - " + err);
+            res.sendStatus(500);
+        } else {
+            res.send(contacts);
```

Comment:
Please change this code to remove the _id from the information that is sent to the user

Reply:
Changes applied!